dxr/query.py

```diff
@@ -189,6 +197,39 @@ def direct_result(self):
                 elif len(results) > 1:
                     return None
 
+    def _perform_search(self, filters, is_line_query, offset, limit, query=None, sort=True):
```

Comment:
Is there a more descriptive name we can give this? To me, it sounds like it has the same semantics as another attr, `es_search`. Maybe mention the highlighting or the things it returns?